runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -1638,6 +1638,276 @@ mod development {
+				assert_ok!(orml_asset_registry::Pallet::<T>::update_asset(
+					<T as frame_system::Config>::RuntimeOrigin::root(),
+					currency_id,
+					None,
+					None,
+					None,
+					None,
+					None,
+					Some(CustomMetadata {
+						// Changed: Allow liquidityPools transferability
+						transferability: CrossChainTransferability::LiquidityPools,
+						mintable: Default::default(),
+						permissioned: Default::default(),
+						// Still allow to be pool currency
+						pool_currency: true,
+					}),
+				));
+				assert_noop!(
+					pallet_liquidity_pools::Pallet::<T>::disallow_investment_currency(
+						RawOrigin::Signed(Keyring::Bob.into()).into(),
+						pool_id,
+						default_tranche_id::<T>(pool_id),
+						currency_id,
+					),
+					pallet_liquidity_pools::Error::<T>::AssetNotLiquidityPoolsWrappedToken
+				);
+
+				// Should fail if currency does not have LiquidityPoolsWrappedToken location in
+				// metadata
+				assert_ok!(orml_asset_registry::Pallet::<T>::update_asset(
+					<T as frame_system::Config>::RuntimeOrigin::root(),
+					currency_id,
+					None,
+					None,
+					None,
+					None,
+					// Changed: Add some location which cannot be converted to
+					// LiquidityPoolsWrappedToken
+					Some(Some(VersionedMultiLocation::V3(Default::default()))),
+					// No change for transferability required as it is already allowed for
+					// LiquidityPools
+					None,
+				));
+				assert_noop!(
+					pallet_liquidity_pools::Pallet::<T>::disallow_investment_currency(
+						RawOrigin::Signed(Keyring::Bob.into()).into(),
+						pool_id,
+						default_tranche_id::<T>(pool_id),
+						currency_id,
+					),
+					pallet_liquidity_pools::Error::<T>::AssetNotLiquidityPoolsWrappedToken
+				);
+
+				// Create new pool for non foreign asset
+				// NOTE: Can be removed after merging https://github.com/centrifuge/centrifuge-chain/pull/1343
+				assert_ok!(orml_asset_registry::Pallet::<T>::register_asset(
+					<T as frame_system::Config>::RuntimeOrigin::root(),
+					AssetMetadata {
+						name: "Acala Dollar".into(),
+						symbol: "AUSD".into(),
+						decimals: 12,
+						location: None,
+						existential_deposit: 1_000_000,
+						additional: CustomMetadata {
+							transferability: Default::default(),
+							mintable: false,
+							permissioned: false,
+							pool_currency: true,
+						},
+					},
+					Some(CurrencyId::AUSD)
+				));
+
+				create_currency_pool::<T>(pool_id + 1, CurrencyId::AUSD, 10_000 * dollar(12));
+
+				// Should fail if currency is not foreign asset
+				assert_noop!(
+					pallet_liquidity_pools::Pallet::<T>::disallow_investment_currency(
+						RawOrigin::Signed(Keyring::Bob.into()).into(),
+						pool_id + 1,
+						// Tranche id is arbitrary in this case, so we don't need to check for the
+						// exact pool_id
+						default_tranche_id::<T>(pool_id + 1),
+						CurrencyId::AUSD,
+					),
+					DispatchError::Token(sp_runtime::TokenError::Unsupported)
+				);
```

Comment:
NIT: Since https://github.com/centrifuge/centrifuge-chain/pull/1343 has been merged, I suppose we can remove this step. Could also do it for other tests as a refactor-creep.

Reply:
Done, there were only 2 tests that were using `CurrencyId::AUSD`, I removed this from both. Awesome catch, thank you!